setup.py

```diff
@@ -41,7 +41,12 @@ def run_tests(self):
                   coverage, fail_under))
             raise SystemExit(1)
 
-VERSION = open('VERSION').read().strip()
+def _get_version():
```

Comment:
Oh, I hate the near-duplication of code here.

Why do we have to move the VERSION into the Python directories anyway?  It isn't a .py file...?